db/seeds.rb

```diff
@@ -272,12 +272,6 @@
   analyses contextuelles et anticiper les modalités de confinement, de remédiation et de
   reconstruction des technologies
   les plus impactantes pour le ministère de la Défense.
-
```

Comment:
why?

Reply:
It's more than 1000 char https://github.com/betagouv/civilsdeladefense/issues/441